packages/desktop-client/src/components/reports/graphs/adjustTextSize.ts

```diff
@@ -0,0 +1,91 @@
+export const adjustTextSize = size => {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏
```suggestion
export const adjustTextSize = (size: number): `${number}px` => {
```

same for the other utils in this file